struqture/src/bosons/bosonic_system.rs

```diff
@@ -53,6 +53,8 @@ pub struct BosonSystem {
     pub(crate) operator: BosonOperator,
```

Comment:
This should also contain the 
#[cfg_attr(feature = "json_schema", derive(schemars::JsonSchema))]
#[cfg_attr(feature = "json_schema", schemars(deny_unknown_fields))]

lines

Reply:
Fixed